server/requirements-dev.txt

```diff
@@ -0,0 +1,2 @@
+pytest
+requests
```

Comment:
I think you might want:
 -r requirements.txt
at the end of this file as well so that travis installs all the requirements, not just these two.

Reply:
`pip install -e .` will bring in the requirements of the package, but I think it's preferable to manage it with with `-r requirements.txt`, as it helps people who are just developing.

Reply:
thanks, added